.github/workflows/pr.yml

```diff
@@ -1,95 +1,195 @@
-name: PullRequest builder TLA+
+name: TLA+ PR Validation
```

Comment:
Can the CI changes be moved to a dedicated PR?

Reply:
They're dependent on this one because of the inclusion of the unicode stuff. I guess I could split them out to their own PR and then rebase this PR on top of that but I guess I question the purpose of all that effort.

Reply:
divide et impera

Reply:
Opened PR here: https://github.com/tlaplus/tlaplus/pull/900